src/it/released-version/invoker.properties

```diff
@@ -0,0 +1,2 @@
+invoker.goals = test
+invoker.buildResult = success
```

Comment:
Maybe a bit picky, but usually files end with a EOL.

Reply:
copied it over from the other ITs, it's the same there
i think this is really cosmetic...

Reply:
Yep. You might laugh, but when I tried to contribute to the Maven checkstyle plugin, those things are mentioned as well. 😅 
Anyway, this does not block anything, just wanted to let you know. LGTM.